rpc-protocol/src/main/java/org/sonarsource/sonarlint/core/rpc/protocol/client/connection/SonarCloudConnectionParams.java

```diff
@@ -21,16 +21,24 @@
 
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
+import org.sonarsource.sonarlint.core.rpc.protocol.common.SonarCloudRegion;
 
 public class SonarCloudConnectionParams {
   private final String organizationKey;
   private final String tokenName;
   private final String tokenValue;
+  private final SonarCloudRegion region;
 
+  @Deprecated(since = "10.14")
   public SonarCloudConnectionParams(String organizationKey, @Nullable String tokenName, @Nullable String tokenValue) {
+    this(organizationKey, tokenName, tokenValue, SonarCloudRegion.EU);
```

Comment:
Here I'm not sure we need a deprecation. Clients are not supposed to call the constructor, they will receive an instance. But maybe for safety we can keep the deprecation. If we keep it, we should document it